bootstrap.php

```diff
@@ -80,27 +62,28 @@ function purge_tables() {
 		function () {
 
 			// Set Global Constants
-			if ( ! defined( 'MODULE_TASKS_VERSION' ) ) {
-				define( 'MODULE_TASKS_VERSION', '0.0.1' );
+			if ( ! defined( 'NFD_MODULE_TASKS_VERSION' ) ) {
+				define( 'NFD_MODULE_TASKS_VERSION', '0.0.1' );
 			}
 
-			if ( ! defined( 'MODULE_TASKS_DIR' ) ) {
-				define( 'MODULE_TASKS_DIR', __DIR__ );
+			if ( ! defined( 'NFD_MODULE_TASKS_DIR' ) ) {
+				define( 'NFD_MODULE_TASKS_DIR', __DIR__ );
 			}
 
-			if ( ! defined( 'MODULE_TASKS_TASK_TABLE_NAME' ) ) {
-				define( 'MODULE_TASKS_TASK_TABLE_NAME', 'nfd_tasks' );
+			if ( ! defined( 'NFD_MODULE_TASKS_TASK_TABLE_NAME' ) ) {
+				define( 'NFD_MODULE_TASKS_TASK_TABLE_NAME', 'nfd_tasks' );
 			}
 
-			if ( ! defined( 'MODULE_TASKS_TASK_RESULTS_TABLE_NAME' ) ) {
-				define( 'MODULE_TASKS_TASK_RESULTS_TABLE_NAME', 'nfd_task_results' );
+			if ( ! defined( 'NFD_MODULE_TASKS_TASK_RESULTS_TABLE_NAME' ) ) {
+				define( 'NFD_MODULE_TASKS_TASK_RESULTS_TABLE_NAME', 'nfd_task_results' );
 			}
 
 			// Make the table setup calls
-			setup_task_table( MODULE_TASKS_TASK_TABLE_NAME );
-			setup_task_results_table( MODULE_TASKS_TASK_RESULTS_TABLE_NAME );
+			setup_task_table( NFD_MODULE_TASKS_TASK_TABLE_NAME );
+			setup_task_results_table( NFD_MODULE_TASKS_TASK_RESULTS_TABLE_NAME );
```

Comment:
@amartya-dev The functions on lines 82 and 83 don't exist anymore and should be removed.